deployments/hadoop-yarn/ansible/51-cephfs-mount.yml

```diff
@@ -31,39 +31,32 @@
 #
 # Requires the following extra params:
 #
-#   mntpath:  '/aglais/data/gaia/dr2'
-#   mntmode:  'rw'
+#   mountpath:   '/aglais/data/gaia/dr2'
+#   mountmode:   'rw'
+#   mountowner:  'fedora'
+#   mountgroup:  'users'
 #
-#   cephuser:  '${cephuser:?}'
-#   cephkey:   '${cephkey:?}'
-#   cephpath:  '${cephpath:?}'
-#   cephnodes: '${cephnodes// /,}'
+#   cephname:  'Jaden-user-share-001-ro'
+#   cephkey:   'AQDWcM9iSFYBNBAApxniJBWuisYXeJFp+m2gXQ=='
```

Comment:
This seems like a test share so I'm assuming no issues with this key appearing here

Reply:
Yep, test data as an example. Good catch though - I should have changed it to something like 'my-example-key'.